snuba/query/processors/tagsmap.py

```diff
@@ -22,6 +26,11 @@ class OptimizableCondition(NamedTuple):
     value: str
 
 
+logger = logging.getLogger("snuba.query-processing")
+
+BEGINNING_OF_TIME = datetime(2019, 12, 11, 0, 0, 0)
```

Comment:
Should this be a parameter on the class constructor?

What is the significance of this date? (This is a rhetorical question — I know it's commented later, but that's a lot further than here.)